api/v1alpha1/k8s_types.go

```diff
@@ -106,6 +105,14 @@ func (v *VolumeSpec) Validate() (ok bool, err error) {
 
 // PodConfigurationSpec contains the configuration for the benchmark pods
 type PodConfigurationSpec struct {
+
+	// Annotations is an unstructured key value map stored with a resource that may be
```

Comment:
The comment should start with the element name: "PodAnnotations is an..." Reference: https://blog.golang.org/godoc-documenting-go-code